app/__init__.py

```diff
@@ -79,8 +80,11 @@ def create_app(debug=False):
                         elif mtype == MachineType.ISPINDEL:
                             active_iSpindel_sessions[uid] = iSpindelSession()
                             active_iSpindel_sessions[uid].alias = aliases[mtype][uid]
+                        elif mtype == MachineType.PICOSTILL:
+                            active_still_sessions[uid] = PicoStillSession()
+                            active_still_sessions[uid].alias = aliases[mtype][uid]

```

Comment:
removing the PicoStill from being mapped as a PicoBrewSession
this will lay the ground work for adding Still session logging (planned to have this month when I get time to finish it)